users/admin.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.utils.translation import ugettext_lazy as _
+
+from users.models import CustomUser
+from users.forms import CustomUserChangeForm, CustomUserCreationForm
+
+class CustomUserAdmin(UserAdmin):
+    fieldsets = (
+        (None, {'fields': ('email', 'password')}),
+        (_('Personal info'), {'fields': ('first_name', 'last_name',
+                                        'date_of_birth', 'phone_number', 'country')}),
+        (_('Status'), {'fields': ('user_type',)}),
+        (_('Permissions'), {'fields': ('is_active', 'is_staff', 'is_superuser',
+                                       'groups', 'user_permissions')}),
+        (_('Important dates'), {'fields': ('last_login',)}),
```

Comment:
I've brainstormed my idea for storing UI text separately with @Karrp and @TheCM and we came up with storing them in separate file(s) in 'constants' folder, which would be potentially located in every app including the base app. 
It's still up to debate how to split them further but for now, you could create 'constants' folder in users with single 'strings.py' file and put string constants in separate classes, for example:
```
from django.utils.translation import ugettext_lazy as _

class CustomUserAdminText:
       PERSONAL_INFO = _('Personal info')
       STATUS = _('Status')
       ...
```
Everyone thinks it's okay?

Reply:
Done